etip/trackers/models.py

```diff
@@ -71,6 +72,15 @@ class Tracker(models.Model):
     def __str__(self):
         return self.name
 
+    def clean_fields(self, exclude=None):
+        super().clean_fields(exclude=exclude)
+        try:
+            re.compile(self.code_signature)
+        except re.error:
+            raise ValidationError(
+                {'code_signature': ["Must be a valid regex."]}
```

Comment:
Why `["Must be a valid regex."]` and not just `"Must be a valid regex."`?
I tested locally and both seem to work the same, I'm just not sure why this is a list.

Reply:
Hum absolutely no reason, I tough it needs to be a list and didn't try without... I'm changing it :)